File: src/main/java/edu/ucf/epoch/epochpatches/EpochEventListeners.java
package edu.ucf.epoch.epochpatches;

import net.neoforged.bus.api.SubscribeEvent;
import net.neoforged.neoforge.event.tick.ServerTickEvent;

public final class EpochEventListeners {
	@SubscribeEvent
	public static void onServerTick(ServerTickEvent.Post evt) {
		EpochPatchesMod.scheduler.runTasks();
	}
	
	
	private EpochEventListeners(){}
}

File: src/main/kotlin/edu/ucf/epoch/epochpatches/EpochEventListeners.kt
package edu.ucf.epoch.epochpatches

import net.neoforged.bus.api.SubscribeEvent
import net.neoforged.neoforge.event.tick.ServerTickEvent

object EpochEventListeners {
	@Suppress("unused")
	@SubscribeEvent
	fun onServerTick(evt: ServerTickEvent.Post) {
		EpochPatchesMod.scheduler.poll()
	}
}